### PR TITLE
GEN-1736 - feat(ImageTextBloc): support RichTextBlock

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

* `ImageTextBlock` now can nest `RichTextBlock`
* Added styles customisations for when `RichTextBlock` as rendered as others block children

## Justify why they are needed

Requested by Peter.

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #3868** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3868?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3868?utm_source=stack-comment).

File: apps/store/src/blocks/RichTextBlock/RichTextBlock.tsx [hunk 1]
```diff
@@ -14,9 +15,13 @@ export type RichTextBlockProps = SbBaseBlockProps<{
...
 }>
 
-export const RichTextBlock = ({ blok }: RichTextBlockProps) => {
+export const RichTextBlock = ({ blok, nested }: RichTextBlockProps) => {
   const content = useMemo(() => renderRichText(blok.content), [blok.content])
 
+  if (nested) {
+    return <NestedRichText largeText={blok.largeText}>{content}</NestedRichText>
+  }
+
```
Comment (gustaveen) [inline apps/store/src/blocks/RichTextBlock/RichTextBlock.tsx:24]: We could us the `ConditionalWrapper` for this as we do it other places where we check if it's a nested block or not :) 

```suggestion
   <ConditionalWrapper
      condition={!nested}
      wrapWith={(children) => (
        <GridLayout.Root>
          <GridLayout.Content
            width={blok.layout?.widths ?? { base: '1', md: '2/3', xl: '1/2' }}
            align={blok.layout?.alignment ?? 'center'}
          >
            {children}
          </GridLayout.Content>
        </GridLayout.Root>
      )}
```

**Review by gustaveen — APPROVED** [review]
Great work! 🙌 Just wondered if we should use a the `conditionalWrapper` for this

**Comment by guilhermespopolin** [comment]
### Merge activity

* **Jan 26, 2:46 AM**: @guilhermespopolin started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3868).

**Pull request merged**